stl/inc/xmemory

```diff
@@ -553,6 +570,17 @@ struct _Normal_allocator_traits { // defines traits for allocators
         }
     }
 
+#if _HAS_CXX23
+    _NODISCARD_RAW_PTR_ALLOC constexpr static allocation_result<pointer, size_type> allocate_at_least(

```

Comment:
No change requested: I observe that this is upgrading plain `_NODISCARD` to `_NODISCARD_RAW_PTR_ALLOC`, which is reasonable (it returns a raw pointer wrapped in a struct, so the message is still applicable).

Reply:
Updated the PR description to mention this improvement.